homeassistant/components/braviatv/manifest.json

```diff
@@ -2,9 +2,9 @@
   "domain": "braviatv",
   "name": "Sony Bravia TV",
   "documentation": "https://www.home-assistant.io/integrations/braviatv",
-  "requirements": ["bravia-tv==1.0.11"],
+  "requirements": ["pybravia==0.1.2"],
```

Comment:
The library traps exceptions now and logs warnings.

https://github.com/Drafteed/pybravia/blob/6dc30cfc41fed5a7b1d1bc6fb8a8bd90737a32d7/pybravia/braviatv.py#L126

It's better to raise library specific exceptions on error instead and let the library user handle error/warning logging. It's easy to spam the log otherwise and it's easier for the library user to be able to take action on error if the library raises an exception. Debug logs are of course always ok.

Reply:
I'd recommend to add type annotations to the library and declare it to be typed with a `py.typed` file.